lib/components/user/places/place-editor.js

```diff
@@ -63,23 +60,28 @@ class PlaceEditor extends Component {
   }
 
   render () {
-    const { errors, handleBlur, handleChange, values: place } = this.props
+    const { errors, handleBlur, handleChange, intl, values: place } = this.props
     const isFixed = isHomeOrWork(place)
     const errorStates = getErrorStates(this.props)
+    const namePlaceholder = intl.formatMessage({id: 'components.PlaceEditor.namePlaceholder'})
 
     return (
       <div>
         {!isFixed && (
         <>
           <FormGroup validationState={errorStates.name} >
             {/* onBlur, onChange, and value are passed automatically. */}
-            <Field aria-label={MESSAGES.SET_PLACE_NAME}
+            <Field aria-label={namePlaceholder}
               as={FormControl}
               name='name'
-              placeholder={MESSAGES.SET_PLACE_NAME}
+              placeholder={namePlaceholder}
             />
             <FormControl.Feedback />
-            {errors.name && <HelpBlock>{errors.name}</HelpBlock>}
+            {errors.name && (
+              <HelpBlock>
+                <FormattedMessage id={errors.name} />
```

Comment:
Please do not use a variable for a message ID. As noted in https://github.com/opentripplanner/otp-react-redux/pull/454/files#r709623526, you could use some kind of identifier to determine the error, but the message ID should still be a hard-coded string.